.github/workflows/release.yml

```
@@ -6,171 +6,224 @@ on:
-        features:
-          - "all_fdws"
-        box:
-          - { runner: blacksmith-4vcpu-ubuntu-2404, arch: amd64 }
-          - { runner: blacksmith-4vcpu-ubuntu-2404-arm, arch: arm64 }
-    runs-on: ${{ matrix.box.runner }}
-    timeout-minutes: 90
-    steps:
-      - name: checkout code
-        uses: actions/checkout@v6
+      - uses: actions/checkout@v6
+      - uses: actions-rs/toolchain@v1
```

actionlint (1.7.11)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)